themes/dark.json

```diff
@@ -450,7 +450,6 @@
 		},
 		{
 			"scope": [
-				"keyword.control",
```

Comment:
Any reason for removing this one?

Reply:
That's because the scope "keyword.control" already shows up in that theme on line 316, and I liked the color there (light blue "#569CD6") vs the one removed on 453 (pinkish "#C586C0"). We can switch it if you like the other color better, but I think we shouldn't have both to avoid bugs and ambiguities. Generally I think each scope should only map to one color.

Reply:
Most of what is in these files come from the themes themselves. I'm surprised you went through them :D